addons/auto-follow-topics/userscript.js

```diff
@@ -0,0 +1,22 @@
+export default async function ({ addon, console, msg }) {
+  const getTopics = () => JSON.parse(sessionStorage.getItem("auto-follow-topics") || "[]");
+  const setTopics = (topics) => sessionStorage.setItem("auto-follow-topics", JSON.stringify(topics));
+  const postError = () => Boolean(document.querySelector(".errorlist"));
+  const topicID = location.href.split("/")[5];
+
+  document.querySelector("[name=AddPostForm]").addEventListener("click", (event) => {
```

Comment:
```suggestion
  document.querySelector("[name=AddPostForm]")?.addEventListener("click", (event) => {
```
Form doesn't exist if user is logged out